PurchasesTests/Networking/DNSCheckerTests.swift

```diff
@@ -19,18 +19,19 @@ import XCTest
 
 class DNSCheckerTests: XCTestCase {
 
-    let apiURL = URL(string: "https://api.revenuecat.com")!
-    let fakeSubscribersURL1 = URL(string: "https://0.0.0.0/subscribers")!
-    let fakeSubscribersURL2 = URL(string: "https://127.0.0.1/subscribers")!
-    let fakeOffersURL = URL(string: "https://0.0.0.0/offers")!
-
-    func testResolvedHost() {
-        let host = DNSChecker.resolvedHost(fromURL: apiURL)
+    private let apiURL = URL(string: "https://api.revenuecat.com")!
+    private let fakeSubscribersURL1 = URL(string: "https://0.0.0.0/subscribers")!
+    private let fakeSubscribersURL2 = URL(string: "https://127.0.0.1/subscribers")!
+    private let fakeOffersURL = URL(string: "https://0.0.0.0/offers")!
+
+    // Note: this test requires a working internet connection
+    func testResolvedHost() throws {
+        let host = try XCTUnwrap(DNSChecker.resolvedHost(fromURL: apiURL))
```

Comment:
perhaps instead of the `// note:`, we can code that into the `XCTUnwrap`?
i.e.:
```suggestion
    func testResolvedHost() throws {
        let host = try XCTUnwrap(DNSChecker.resolvedHost(fromURL: apiURL), message: "The host couldn't be resolved. Note that this test requires a working internet connection")
```

Reply:
Much better, changed! _(I changed it manually because of the long line)_